diff --git a/CMakeLists.txt b/CMakeLists.txt
index ee94f5d..bacf902 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,7 +141,7 @@ message(STATUS "Check for M4 macro processor: found ${M4}")
 
 ###############################################################################
 
-add_subdirectory(doc)
+# add_subdirectory(doc)
 add_subdirectory(runtime)
 add_subdirectory(scripts)
 add_subdirectory(src)
diff --git a/runtime/CMakeLists.txt b/runtime/CMakeLists.txt
index 10ef48f..4b0c5ca 100644
--- a/runtime/CMakeLists.txt
+++ b/runtime/CMakeLists.txt
@@ -32,8 +32,8 @@
 cmake_minimum_required(VERSION 3.16)
 project(csmith_runtime)
 
-set(INCLUDE_DIR "${CMAKE_INSTALL_PREFIX}/include")
-set(LIB_DIR     "${CMAKE_INSTALL_PREFIX}/lib")
+set(INCLUDE_DIR "${CMAKE_INSTALL_PREFIX}/include/csmith")
+set(LIB_DIR     "${CMAKE_INSTALL_PREFIX}/lib64")
 
 ###############################################################################
 
